sdk/src/test/scala/com/horizen/fixtures/MockedSidechainNodeViewHolderFixture.scala

```diff
@@ -11,7 +11,7 @@ class MockedSidechainNodeViewHolder(sidechainSettings: SidechainSettings,
                                     state: SidechainState,
                                     wallet: SidechainWallet,
                                     mempool: SidechainMemoryPool)
-  extends SidechainNodeViewHolder(sidechainSettings, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null) {
+  extends SidechainNodeViewHolder(sidechainSettings, null, null, null, null, null, null, null, null, null, null, null,null, null, null, null) {
```

Comment:
Cosmetic: probably some modification has been reverted here, just a blank is missing here